app/apollo/resolvers/serviceSubscription.js

```diff
@@ -83,21 +83,24 @@ const serviceResolvers = {
 
       logger.debug({req_id, user, orgId }, `${queryName} enter`);
 
-      try {
-        await validAuth(me, orgId, ACTIONS.READ, TYPES.SERVICESUBSCRIPTION, queryName, context);
+      await validAuth(me, orgId, ACTIONS.READ, TYPES.SERVICESUBSCRIPTION, queryName, context);
```

Comment:
Why did you move this our of the try/catch block? Does it land in the right place if validAuth throws?